src/components/availability-label/availability-label.tsx

```diff
@@ -91,6 +91,7 @@ export const AvailabilityLabel: React.FC<AvailabilityLabelProps> = ({
       role="button"
       tabIndex={0}
       data-cy={dataCy}
+      aria-pressed={selected}
     >
       <div className={classes.triangle} />
       <img className={classes.check} src={CheckIcon} alt="check-icon" />
```

Comment:
Considering this bug report I think two more changes are needed:

1. The `alt` attribute for the `CheckIcon` should be removed. It is included in the screen reader output as shown by the bug report and it is currently not translatable. Any information which could be provided using the icon is better represented through `aria-pressed` so we might as well remove it entirely.
2. The original change and the change in 1. must be reflected in the design system which also have a story for selected availability labels.